Containers/talk/Dockerfile

```diff
@@ -1,18 +1,19 @@
 FROM nats:2.9.18-scratch as nats
 FROM strukturag/nextcloud-spreed-signaling:1.1.2 as signaling
-FROM coturn/coturn:4.6.2-r3-alpine
-USER root
-# Pin alpine version manually as long as https://github.com/coturn/coturn/issues/1226 is not done
-ENV ALPINE_VERSION=3.18
+FROM ghcr.io/processone/eturnal:x.x.x-otpxx AS eturnal
+FROM alpine:3.18.2
 
-COPY --from=nats /nats-server /usr/local/bin/nats-server
+ENV STUN_SERVICE="stun.nextcloud.com 443"
+
+COPY --from=eturnal   /opt/eturnal                        /opt/eturnal
+COPY --from=eturnal   /usr/local/bin/eturnalctl           /usr/local/bin/eturnalctl
```

Comment:
You removed the sym-link for `/opt/eturnal/bin/stun`. So either put back the `ln -s /opt/eturnal/bin/stun /usr/local/bin/stun` or `COPY --from=eturnal /usr/local/bin/stun /usr/local/bin/stun`

`eturnalctl` will look for `stun` in `$PATH` for the STUN query.

Reply:
done